frontend/apps/reader/modules/readergesture.lua

```diff
@@ -1302,13 +1307,29 @@ function ReaderGesture:gestureAction(action, ges)
             self.ui:handleEvent(Event:new("ShowFlDialog"))
         end
     elseif action == "increase_frontlight" then
+        -- when using frontlight system settings
+        if lightFrontlight() then
+            UIManager:show(Notification:new{
+                text = _("Frontlight controlled by system settings."),
```

Comment:
is/will now be?

Also I think 1 s might be a bit short.

Reply:
2 s or more?

Reply:
I'd say 2 or 3.

Reply:
2.5 s :)